wnfs-wasm/Cargo.toml

```diff
@@ -25,7 +25,7 @@ console_error_panic_hook = { version = "0.1", optional = true }
 getrandom = { version = "0.2", features = ["js"] }
 js-sys = { version = "0.3", optional = true }
 rand_core = "0.6"
-wasm-bindgen = { version = "0.2", optional = true, features = ["serde-serialize"] }
+wasm-bindgen = { version = "0.2.84", optional = true, features = ["serde-serialize"] }
```

Comment:
Does having the minor version affect anything?

Reply:
Yes, there's some important bugfixes in 0.2.84: https://github.com/rustwasm/wasm-bindgen/pull/3117